epxresults/job.py

```diff
@@ -364,36 +360,179 @@ def get_job_variable_table(
 
         # check if variable is available
         if variable not in self.global_variables:
-            msg = (f"{variable} is not available to load. See "
-                   "`self.global_variables` for a list of "
-                   "available variables.")
+            msg = (
+                f"{variable} is not available to load. See "
+                "`self.global_variables` for a list of "
+                "available variables."
+            )
             raise ValueError(msg)
 
         fname = f"FRED.{variable}.csv"
         path = os.path.join(
-            self.path_to_job,
-            self._interval_path_component(interval),
-            fname
+            self.path_to_job, self._interval_path_component(interval), fname
         )
 
+        df = self._read_plot_data_file(path, interval)
+        self._validate_table_format(format)
+        if format == "long":
+            df = self._convert_wide_plot_table_to_long(df, variable)
+
+        return df
+
+    def get_job_rt_table(
+        self, condition: str, interval: OutputInterval = "daily", format: str = "long"
+    ) -> pd.DataFrame:
+        """Time series of the reproductive rate, R(t), of the given
+        transmissible condition.
+
+        See `FRED documentation`_ for further details of the definition of this
+        quantity.
+
+        .. _FRED documentation: https://docs.epistemix.com/projects/fred-sims/en/latest/sim-guide/chapter2.html#reproductive-rate-rt
+
+        Examples
+        --------
+        >>> job = FREDJob(job_key='simpleflu')
+        >>> df = job.get_job_rt_table('INF').head()
+        >>> df  # doctest: +SKIP
```

Comment:
I assume `# doctest: +SKIP` makes pytest skip this line?  If so, this is excellent!

Reply:
Yes, that's it. The issue here is that I can't guarantee identical numerical outputs when someone else runs the scripts to generate the test data. This lets us show the structure of the generated table in the docstring without needing to worry about making such assurances.